applications/nublado/templates/controller-ingress-files.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: gafaelfawr.lsst.io/v1alpha1
+kind: GafaelfawrIngress
+metadata:
+  name: "nublado-controller-files"
+  labels:
+    {{- include "nublado.labels" . | nindent 4 }}
+config:
+  baseUrl: {{ .Values.global.baseUrl | quote }}
+  delegate:
+    internal:
+      service: "nublado"
+      scopes: []
```

Sorry, apparently that wasn't clear. I didn't mean to delete the scopes restriction. This would be syntactically invalid and will be rejected by the CRD. You have to specify both (a) the scope required to access this service and (b) the type of token to delegate to it.

In other words:
```suggestion
  scopes:
    all:
      - "exec:notebook"
  delegate:
    internal:
      service: "nublado"
      scopes: []
```